src/backup_camera.h

```diff
@@ -25,11 +25,14 @@ extern "C" {
 #include "WiringPiButtons.hpp"
 #endif
 
-//#include "MusicBar.h"
-//#include "SongPlayer.h"
+
 #include "graphics_handler.h"
 #include "videoStream.hpp"
 
+//event flags to pass back to main super loop
+#define QUIT_EVENT_FLAG 'A'
+#define ENTER_FULLSCREEN_EVENT_FLAG 'B'
+#define EXIT_FULLSCREEN_EVENT_FLAG 'C'
```

Comment:
`#defines` are not generally used in c++ due to the cryptic compiler warnings they give. You should use constants encapsulated in a namespace.